ServletException - Could not locate ordinal parameter [0], expecting one of [1, 2, 3]

**Summary:** 

After bumping Hibernate dependency in both pom.xml files to:

```
<dependency>
  <groupId>org.hibernate</groupId>
  <artifactId>hibernate-core</artifactId>
  <version>5.6.7.Final</version>
</dependency>
```

...the webapp starts up ok, but when it tries to show results in the web UI I see this error when the website hits the REST API endpoint:

```
[qtp2025322501-26] WARN org.eclipse.jetty.server.HttpChannel - /api/gtfs-rt-feed/monitor-data/1
javax.servlet.ServletException: java.lang.IllegalArgumentException: Could not locate ordinal parameter [0], expecting one of [1, 2, 3]
	at org.glassfish.jersey.servlet.WebComponent.service(WebComponent.java:423)
	at org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:386)
	at org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:334)
	at org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:221)
	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:799)
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:550)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:233)
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1435)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:188)
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:501)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:186)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1350)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127)
	at org.eclipse.jetty.server.Server.handle(Server.java:516)
	at org.eclipse.jetty.server.HttpChannel.lambda$handle$1(HttpChannel.java:388)
	at org.eclipse.jetty.server.HttpChannel.dispatch(HttpChannel.java:633)
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:380)
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:277)
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:311)
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:105)
	at org.eclipse.jetty.io.ChannelEndPoint$1.run(ChannelEndPoint.java:104)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:336)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:313)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:171)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:129)
	at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:383)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:882)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1036)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.IllegalArgumentException: Could not locate ordinal parameter [0], expecting one of [1, 2, 3]
	at org.hibernate.query.internal.ParameterMetadataImpl.getOrdinalParameterDescriptor(ParameterMetadataImpl.java:154)
	at org.hibernate.query.internal.ParameterMetadataImpl.getQueryParameter(ParameterMetadataImpl.java:204)
	at org.hibernate.query.internal.QueryParameterBindingsImpl.getBinding(QueryParameterBindingsImpl.java:183)
	at org.hibernate.query.internal.AbstractProducedQuery.setParameter(AbstractProducedQuery.java:519)
	at org.hibernate.query.internal.NativeQueryImpl.setParameter(NativeQueryImpl.java:656)
	at org.hibernate.query.internal.NativeQueryImpl.setParameter(NativeQueryImpl.java:66)
	at edu.usf.cutr.gtfsrtvalidator.api.resource.GtfsRtFeed.getMonitorData(GtfsRtFeed.java:200)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.glassfish.jersey.server.model.internal.ResourceMethodInvocationHandlerFactory$1.invoke(ResourceMethodInvocationHandlerFactory.java:81)
	at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher$1.run(AbstractJavaResourceMethodDispatcher.java:144)
	at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.invoke(AbstractJavaResourceMethodDispatcher.java:161)
	at org.glassfish.jersey.server.model.internal.JavaResourceMethodDispatcherProvider$ResponseOutInvoker.doDispatch(JavaResourceMethodDispatcherProvider.java:160)
	at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.dispatch(AbstractJavaResourceMethodDispatcher.java:99)
	at org.glassfish.jersey.server.model.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:389)
	at org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:347)
	at org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:102)
	at org.glassfish.jersey.server.ServerRuntime$2.run(ServerRuntime.java:308)
	at org.glassfish.jersey.internal.Errors$1.call(Errors.java:271)
	at org.glassfish.jersey.internal.Errors$1.call(Errors.java:267)
	at org.glassfish.jersey.internal.Errors.process(Errors.java:315)
	at org.glassfish.jersey.internal.Errors.process(Errors.java:297)
	at org.glassfish.jersey.internal.Errors.process(Errors.java:267)
	at org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:317)
	at org.glassfish.jersey.server.ServerRuntime.process(ServerRuntime.java:291)
	at org.glassfish.jersey.server.ApplicationHandler.handle(ApplicationHandler.java:1140)
	at org.glassfish.jersey.servlet.WebComponent.service(WebComponent.java:403)
	... 29 more
```

**Steps to reproduce:** 

Update dependencies to:

```
<dependency>
  <groupId>org.hibernate</groupId>
  <artifactId>hibernate-core</artifactId>
  <version>5.6.7.Final</version>
</dependency>
```

See example in PR https://github.com/MobilityData/gtfs-realtime-validator/pull/140.

**Expected behavior:** 

Everything should work as in main branch - website should show validation results

**Observed behavior:** 

Validation results are never shown in the website, and the console contains the above stacktrace.

**Platform:** 

Windows 10 and JDK 17.0.2

If I run `mvn test`, I also see this test failure:

```
[main] INFO edu.usf.cutr.gtfsrtvalidator.db.GTFSDB - Table initialized successfully
[ERROR] Tests run: 2, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 1.127 s <<< FAILURE! - in edu.usf.cutr.gtfsrtvalidator.test.queries.QueryTest
[ERROR] testViewErrorLogModel(edu.usf.cutr.gtfsrtvalidator.test.queries.QueryTest)  Time elapsed: 0.619 s  <<< ERROR!
java.lang.IllegalArgumentException: Could not locate ordinal parameter [0], expecting one of [1, 2, 3]
        at edu.usf.cutr.gtfsrtvalidator.test.queries.QueryTest.testViewErrorLogModel(QueryTest.java:250)

[ERROR] testViewErrorSummaryModel(edu.usf.cutr.gtfsrtvalidator.test.queries.QueryTest)  Time elapsed: 0.507 s  <<< ERROR!
java.lang.IllegalArgumentException: Could not locate ordinal parameter [0], expecting one of [1, 2, 3]
        at edu.usf.cutr.gtfsrtvalidator.test.queries.QueryTest.testViewErrorSummaryModel(QueryTest.java:146)

[INFO] 
[INFO] Results:
[INFO]
[ERROR] Errors: 
[ERROR]   QueryTest.testViewErrorLogModel:250 » IllegalArgument Could not locate ordinal...
[ERROR]   QueryTest.testViewErrorSummaryModel:146 » IllegalArgument Could not locate ord...
[INFO]
[ERROR] Tests run: 3, Failures: 0, Errors: 2, Skipped: 0
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for gtfs-realtime-validator 1.0.0-SNAPSHOT:
[INFO]
[INFO] gtfs-realtime-validator ............................ SUCCESS [  0.005 s]
[INFO] gtfs-realtime-validator-lib ........................ SUCCESS [ 20.266 s]
[INFO] gtfs-realtime-validator-webapp ..................... FAILURE [01:46 min]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  02:07 min
[INFO] Finished at: 2022-03-23T16:48:42-04:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.0.0-M3:test (default-test) on project gtfs-realtime-validator-webapp: There are test failures.
```

--- a/gtfs-realtime-validator-lib/pom.xml
+++ b/gtfs-realtime-validator-lib/pom.xml
@@ -92,7 +92,7 @@
         <dependency>
             <groupId>org.hibernate</groupId>
             <artifactId>hibernate-core</artifactId>
-            <version>5.2.18.Final</version>
+            <version>5.6.7.Final</version>
         </dependency>
         <dependency>
             <groupId>org.javassist</groupId>

--- a/gtfs-realtime-validator-lib/src/main/java/edu/usf/cutr/gtfsrtvalidator/lib/model/ViewErrorLogModel.java
+++ b/gtfs-realtime-validator-lib/src/main/java/edu/usf/cutr/gtfsrtvalidator/lib/model/ViewErrorLogModel.java
@@ -25,7 +25,7 @@ import java.io.Serializable;
 @Entity
 @NamedNativeQuery(name = "ErrorLogByrtfeedID",
         query = // Retrieve the remaining columns, title and severity from Error and FinalResult tables on matching errorIds.
-                "SELECT rowIdentifier, ? AS rtFeedID, errorId AS id, " +
+                "SELECT rowIdentifier, :gtfsRtId1 AS rtFeedID, errorId AS id, " +
                     "Error.title, Error.severity, iterationId, occurrence, loggingTime " +
                 "FROM Error " +
                 "INNER JOIN " +
@@ -50,9 +50,9 @@ import java.io.Serializable;
                                     "INNER JOIN " +
                                     "(SELECT  IterationID, IterationTimestamp, feedTimestamp " +
                                     "FROM GtfsRtFeedIteration " +
-                                    "WHERE rtFeedID = ?) GtfsRtFeedIDIteration " +
+                                    "WHERE rtFeedID = :gtfsRtId2) GtfsRtFeedIDIteration " +
                                     "ON MessageLog.iterationID = GtfsRtFeedIDIteration.IterationID " +
-                                        "AND IterationTimestamp >= ? AND IterationTimestamp <= ? " +
+                                        "AND IterationTimestamp >= :sessionStartTime AND IterationTimestamp <= :sessionEndTime " +
                                 ") errorLog " +
                             "ORDER BY IterationID " +
                             ") " +

--- a/gtfs-realtime-validator-lib/src/main/java/edu/usf/cutr/gtfsrtvalidator/lib/model/ViewErrorSummaryModel.java
+++ b/gtfs-realtime-validator-lib/src/main/java/edu/usf/cutr/gtfsrtvalidator/lib/model/ViewErrorSummaryModel.java
@@ -24,7 +24,7 @@ import java.io.Serializable;
 @XmlRootElement
 @Entity
 @NamedNativeQuery(name = "ErrorSummaryByrtfeedID",
-    query = "SELECT ? AS rtFeedID, errorID AS id, " +
+    query = "SELECT :gtfsRtId1 AS rtFeedID, errorID AS id, " +
                 "title, severity, totalCount, lastTime, " +
                 "lastFeedTime, lastIterationId, lastRowId " +
             "FROM Error " +
@@ -51,9 +51,9 @@ import java.io.Serializable;
                                 "INNER JOIN " +
                                 "(SELECT  IterationID, IterationTimestamp, feedTimestamp " +
                                 "FROM GtfsRtFeedIteration " +
-                                "WHERE rtFeedID = ?) GtfsRtFeedIDIteration " +
+                                "WHERE rtFeedID = :gtfsRtId2) GtfsRtFeedIDIteration " +
                             "ON MessageLog.iterationID = GtfsRtFeedIDIteration.IterationID " +
-                                "AND IterationTimestamp >= ? AND IterationTimestamp <= ? " +
+                                "AND IterationTimestamp >= :sessionStartTime AND IterationTimestamp <= :sessionEndTime " +
                             ") errorLog " +
                             "ORDER BY iterationId " +
                         ") " +

--- a/gtfs-realtime-validator-lib/src/main/java/edu/usf/cutr/gtfsrtvalidator/lib/model/ViewFeedIterationsCount.java
+++ b/gtfs-realtime-validator-lib/src/main/java/edu/usf/cutr/gtfsrtvalidator/lib/model/ViewFeedIterationsCount.java
@@ -28,8 +28,8 @@ import javax.xml.bind.annotation.XmlRootElement;
 @NamedNativeQuery(name = "feedIterationsCount",
         query = "SELECT count(IterationID) AS iterationCount " +
                 "FROM GtfsRtFeedIteration " +
-                "WHERE rtFeedID = ? " +
-                    "AND IterationTimestamp >= ? AND IterationTimestamp <= ? ",
+                "WHERE rtFeedID = :gtfsRtId " +
+                    "AND IterationTimestamp >= :sessionStartTime AND IterationTimestamp <= :sessionEndTime ",
         resultClass = ViewFeedIterationsCount.class)
 public class ViewFeedIterationsCount {
 

--- a/gtfs-realtime-validator-lib/src/main/java/edu/usf/cutr/gtfsrtvalidator/lib/model/ViewFeedMessageModel.java
+++ b/gtfs-realtime-validator-lib/src/main/java/edu/usf/cutr/gtfsrtvalidator/lib/model/ViewFeedMessageModel.java
@@ -31,12 +31,12 @@ import java.io.InputStream;
     @NamedNativeQuery (name = "feedMessageByIterationId",
             query = "SELECT feedProtobuf AS feedMessage " +
                     "FROM GtfsRtFeedIteration " +
-                    "WHERE IterationID = ? ",
+                    "WHERE IterationID = :iterationId ",
             resultClass = ViewFeedMessageModel.class),
     @NamedNativeQuery (name = "feedMessageByGtfsRtId",
             query = "SELECT feedProtobuf AS feedMessage " +
                     "FROM GtfsRtFeedIteration " +
-                    "WHERE rtFeedId = ?  ORDER BY IterationTimestamp DESC",
+                    "WHERE rtFeedId = :gtfsRtId  ORDER BY IterationTimestamp DESC",
             resultClass = ViewFeedMessageModel.class)
 })
 public class ViewFeedMessageModel {

--- a/gtfs-realtime-validator-lib/src/main/java/edu/usf/cutr/gtfsrtvalidator/lib/model/ViewFeedUniqueResponseCount.java
+++ b/gtfs-realtime-validator-lib/src/main/java/edu/usf/cutr/gtfsrtvalidator/lib/model/ViewFeedUniqueResponseCount.java
@@ -28,8 +28,8 @@ import javax.xml.bind.annotation.XmlRootElement;
 @NamedNativeQuery(name = "feedUniqueResponseCount",
         query = "SELECT count(*) AS uniqueFeedCount " +
                 "FROM GtfsRtFeedIteration " +
-                "WHERE (rtFeedID = ? " +
-                    "AND IterationTimestamp >= ? AND IterationTimestamp <= ? " +
+                "WHERE (rtFeedID = :gtfsRtId " +
+                    "AND IterationTimestamp >= :sessionStartTime AND IterationTimestamp <= :sessionEndTime " +
                     "AND feedProtobuf IS NOT NULL) ",
         resultClass = ViewFeedUniqueResponseCount.class)
 public class ViewFeedUniqueResponseCount {

--- a/gtfs-realtime-validator-lib/src/main/java/edu/usf/cutr/gtfsrtvalidator/lib/model/ViewGtfsRtFeedErrorCountModel.java
+++ b/gtfs-realtime-validator-lib/src/main/java/edu/usf/cutr/gtfsrtvalidator/lib/model/ViewGtfsRtFeedErrorCountModel.java
@@ -36,9 +36,9 @@ import javax.xml.bind.annotation.XmlRootElement;
                 "INNER JOIN " +
                 "(SELECT IterationID, IterationTimestamp " +
                 "FROM GtfsRtFeedIteration " +
-                "WHERE rtFeedID = ?) GtfsRtFeedIDIteration " +
+                "WHERE rtFeedID = :gtfsRtId) GtfsRtFeedIDIteration " +
                 "ON MessageLog.iterationID = GtfsRtFeedIDIteration.IterationID " +
-                    "AND IterationTimestamp >= ? AND IterationTimestamp <= ? " +
+                    "AND IterationTimestamp >= :sessionStartTime AND IterationTimestamp <= :sessionEndTime " +
                 "GROUP BY errorID) errorCount " +
                 "ON Error.errorID = errorCount.errorID ",
         resultClass = ViewGtfsRtFeedErrorCountModel.class)

--- a/gtfs-realtime-validator-lib/src/main/java/edu/usf/cutr/gtfsrtvalidator/lib/model/ViewIterationErrorsModel.java
+++ b/gtfs-realtime-validator-lib/src/main/java/edu/usf/cutr/gtfsrtvalidator/lib/model/ViewIterationErrorsModel.java
@@ -35,9 +35,9 @@ import javax.xml.bind.annotation.XmlRootElement;
                     "INNER JOIN " +
                         "(SELECT messageId, errorId " +
                         "FROM MessageLog " +
-                        "WHERE iterationId = ?) MessageLogIteration " +
+                        "WHERE iterationId = :iterationId) MessageLogIteration " +
                     "ON Occurrence.messageId = MessageLogIteration.messageId " +
-                    "WHERE messageId = ? ) OccurrenceList " +
+                    "WHERE messageId = :messageId ) OccurrenceList " +
                 "ON Error.errorId = OccurrenceList.errorId " +
                 "ORDER BY occurrenceId ",
         resultClass = ViewIterationErrorsModel.class)

--- a/gtfs-realtime-validator-webapp/pom.xml
+++ b/gtfs-realtime-validator-webapp/pom.xml
@@ -139,7 +139,7 @@
         <dependency>
             <groupId>org.hibernate</groupId>
             <artifactId>hibernate-core</artifactId>
-            <version>5.2.18.Final</version>
+            <version>5.6.7.Final</version>
         </dependency>
         <dependency>
             <groupId>org.javassist</groupId>

--- a/gtfs-realtime-validator-webapp/src/main/java/edu/usf/cutr/gtfsrtvalidator/api/resource/GtfsRtFeed.java
+++ b/gtfs-realtime-validator-webapp/src/main/java/edu/usf/cutr/gtfsrtvalidator/api/resource/GtfsRtFeed.java
@@ -197,35 +197,35 @@ public class GtfsRtFeed {
 
         ViewFeedIterationsCount iterationsCount;
         iterationsCount = (ViewFeedIterationsCount) session.createNamedQuery("feedIterationsCount", ViewFeedIterationsCount.class)
-                .setParameter(0, gtfsRtId)
-                .setParameter(1, sessionStartTime)
-                .setParameter(2, sessionEndTime)
+                .setParameter("gtfsRtId", gtfsRtId)
+                .setParameter("sessionStartTime", sessionStartTime)
+                .setParameter("sessionEndTime", sessionEndTime)
                 .uniqueResult();
         mergeMonitorData.setIterationCount(iterationsCount.getIterationCount());
 
         ViewFeedUniqueResponseCount uniqueResponseCount;
         uniqueResponseCount = (ViewFeedUniqueResponseCount) session.createNamedQuery("feedUniqueResponseCount", ViewFeedUniqueResponseCount.class)
-                .setParameter(0, gtfsRtId)
-                .setParameter(1, sessionStartTime)
-                .setParameter(2, sessionEndTime)
+                .setParameter("gtfsRtId", gtfsRtId)
+                .setParameter("sessionStartTime", sessionStartTime)
+                .setParameter("sessionEndTime", sessionEndTime)
                 .uniqueResult();
         mergeMonitorData.setUniqueFeedCount(uniqueResponseCount.getUniqueFeedCount());
 
         List<ViewGtfsRtFeedErrorCountModel> viewGtfsRtFeedErrorCountModel;
         viewGtfsRtFeedErrorCountModel = session.createNamedQuery("feedErrorCount", ViewGtfsRtFeedErrorCountModel.class)
-                .setParameter(0, gtfsRtId)
-                .setParameter(1, sessionStartTime)
-                .setParameter(2, sessionEndTime)
+                .setParameter("gtfsRtId", gtfsRtId)
+                .setParameter("sessionStartTime", sessionStartTime)
+                .setParameter("sessionEndTime", sessionEndTime)
                 .list();
         mergeMonitorData.setViewGtfsRtFeedErrorCountModelList(viewGtfsRtFeedErrorCountModel);
 
         List<ViewErrorSummaryModel> feedSummary;
         int fromRow = (summaryCurPage - 1) * summaryRowsPerPage;
         feedSummary = session.createNamedQuery("ErrorSummaryByrtfeedID", ViewErrorSummaryModel.class)
-                .setParameter(0, gtfsRtId)
-                .setParameter(1, gtfsRtId)
-                .setParameter(2, sessionStartTime)
-                .setParameter(3, sessionEndTime)
+                .setParameter("gtfsRtId1", gtfsRtId)
+                .setParameter("gtfsRtId2", gtfsRtId)
+                .setParameter("sessionStartTime", sessionStartTime)
+                .setParameter("sessionEndTime", sessionEndTime)
                 .setFirstResult(fromRow)
                 .setMaxResults(summaryRowsPerPage)
                 .list();
@@ -245,10 +245,10 @@ public class GtfsRtFeed {
         // Getting the value of fromRow from the rowsPerPage and currentPage values.
         fromRow = (logCurPage - 1) * logRowsPerPage;
         feedLog = session.createNamedQuery("ErrorLogByrtfeedID", ViewErrorLogModel.class)
-                .setParameter(0, gtfsRtId)
-                .setParameter(1, gtfsRtId)
-                .setParameter(2, sessionStartTime)
-                .setParameter(3, sessionEndTime)
+                .setParameter("gtfsRtId1", gtfsRtId)
+                .setParameter("gtfsRtId2", gtfsRtId)
+                .setParameter("sessionStartTime", sessionStartTime)
+                .setParameter("sessionEndTime", sessionEndTime)
                 .setParameterList("errorIds", removeIds)
                 .setFirstResult(fromRow)
                 .setMaxResults(logRowsPerPage)
@@ -279,11 +279,11 @@ public class GtfsRtFeed {
          Session session = GTFSDB.initSessionBeginTrans();
          if(iterationId != -1) {
              feedMessageModel = session.createNamedQuery("feedMessageByIterationId", ViewFeedMessageModel.class)
-                 .setParameter(0, iterationId)
+                 .setParameter("iterationId", iterationId)
                  .uniqueResult();
          } else {
              feedMessageModel = session.createNamedQuery("feedMessageByGtfsRtId", ViewFeedMessageModel.class)
-                     .setParameter(0, gtfsRtId)
+                     .setParameter("gtfsRtId", gtfsRtId)
                      .setMaxResults(1).getSingleResult();
          }
 
@@ -329,8 +329,8 @@ public class GtfsRtFeed {
         for (int messageId: messageIdList) {
             session = GTFSDB.initSessionBeginTrans();
             viewIterationErrorsModelList = session.createNamedQuery("IterationIdErrors", ViewIterationErrorsModel.class)
-                    .setParameter(0, iterationId)
-                    .setParameter(1, messageId)
+                    .setParameter("iterationId", iterationId)
+                    .setParameter("messageId", messageId)
                     .list();
             GTFSDB.closeSession(session);
             if (!viewIterationErrorsModelList.isEmpty()) {

